### PR TITLE
[TW-1403] Doc new filter options for Postman proxy

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Update proxy page with new filter options
- Update Interceptor page with new results UX

View the beta docs here:
- [Capture traffic using the Postman built-in proxy](https://tw-1403-proxy-filter-options.learning.postman-beta.com/docs/sending-requests/capturing-request-data/capture-with-proxy/)
- [Capture traffic from a web browser using Postman Interceptor](https://tw-1403-proxy-filter-options.learning.postman-beta.com/docs/sending-requests/capturing-request-data/interceptor/)

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Comment by VShingala** [comment]
@mark-anthony-postman The Beta docs mentioned doesn't seem to be available for me. I tried with and without VPN but same result.

**Comment by mark-anthony-postman** [comment]
> @mark-anthony-postman The Beta docs mentioned doesn't seem to be available for me. I tried with and without VPN but same result.

@VShingala It looks like our beta docs deployments aren't working at the moment. Apologies for that! I'll let you know when they beta docs are back up.

**Review by VShingala — COMMENTED** [review]
Added one improvement to provide a bit more context. Rest looks good!

File: src/pages/docs/sending-requests/capturing-request-data/capture-with-proxy.md [hunk 1]
```diff
@@ -81,22 +81,24 @@ From the proxy session tab, you can take the following actions:
...
 * Select **Save responses for requests** to save each request's response along with the captured request.
 
 * Select **Capture cookies** if you want to capture cookies along with requests during the proxy session.
 
     > You can also capture cookies and sync them to Postman without starting a proxy session. Learn more about [syncing cookies](/docs/sending-requests/capturing-request-data/syncing-cookies/).
 
-<img alt="Control a proxy session" src="https://assets.postman.com/postman-docs/v10/proxy-control-session-v10-18.jpg" />
+* Select the sort icon <img alt="Column sort icon" src="https://assets.postman.com/postman-docs/v10/icon-column-sort.jpg#icon" width="12px"> in the column headers to sort the results in ascending or descending order.
+
+* Select the filter icon <img alt="Filter icon" src="https://assets.postman.com/postman-docs/icon-filter.jpg#icon" width="16px"> in the column headers to filter the requests and responses captured during the session:
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/capturing-request-data/capture-with-proxy.md]: @VShingala Yes, great suggestion! This concept had definitely gotten lost. I've updated the description of the filters in section "Step 2: Control the proxy session". Then I have a briefer mention of using the filters after stopping the session in "Step 4: Stop the proxy session"

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md [hunk 2]
```diff
@@ -91,22 +91,28 @@ You can work with the Interceptor session results in the following ways:
 * To rename the Interceptor session, select the name **Interceptor debug session** and enter a new name.
 * Select the **Requests** tab to view information about incoming requests and responses, or select the **Cookies** tab to view information about captured cookies.
 * Use the search box to find specific requests or cookies.
-* Select **>** next to a request to view more details.
+* Select the sort icon <img alt="Column sort icon" src="https://assets.postman.com/postman-docs/v10/icon-column-sort.jpg#icon" width="12px"> in the column headers to sort the results in ascending or descending order.
+* Select the filter icon <img alt="Filter icon" src="https://assets.postman.com/postman-docs/icon-filter.jpg#icon" width="16px"> in the column headers to filter the requests and responses captured during the session:
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/capturing-request-data/interceptor.md]: @VShingala I added the updated filter information in the "Starting an Interceptor session" section. Then I have the briefer mention of using the filters after the session is stopped in "Viewing Interceptor results in Postman".

**Review by VShingala — APPROVED** [review]
Changes looks good!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**